.github/workflows/ingest-test-fixtures-update-pr.yml

```diff
@@ -0,0 +1,110 @@
+name: Ingest Test Fixtures Update PR
+
+on:
+  push:
```

Comment:
does this mean it gets triggered for every push?

Reply:
it does, but it will get cancelled if the commit message doesn't contain "ingest-test-fixtures-update". I couldn't find a better way to do trigger this on commit, but totally open if you have other thoughts?

Reply:
ah, i see. works for now!